### PR TITLE
Updated Calculator to latest version of RNW

### PR DESCRIPTION
**Pull request by jonthysell**
Steps I used:

1. New branch

```
git checkout -b updatecalc
```

2. New app with the CLI:

```
cd samples
rmdir /s /q Calculator
yarn global add react-native-cli
react-native init Calculator --version ^0.60.0
cd Calculator
yarn add rnpm-plugin-windows
react-native windows
```

`Yes` to replace metro.config.js.

3. Restore original app code:

```
del App.js
git restore App.tsx
```

4. Verify it works:

```
react-native run-windows
```

5. Restore and update README:

```
git restore README.md
```

6. Added `*.binlog` to .gitignore.

7. Create update commit:

```
git add .
git commit
```

### CODE REVIEW

File: samples/Calculator/windows/Calculator/MainPage.h [hunk 1]
```diff
@@ -1,25 +0,0 @@
-#pragma once
-
-#include "MainPage.g.h"
-
-
-
-namespace winrt::Calculator::implementation
```
[the remark this answers is not in the document]
Comment (jonthysell) [reply samples/Calculator/windows/Calculator/MainPage.h]: The template doesn't create a main page anymore, the ReactApplication just adds the react control to the root of the xaml app.

[the remark this answers is not in the document]
Comment (jonthysell) [reply samples/Calculator/windows/Calculator/MainPage.h]: wait a second, this is the old template

File: samples/Calculator/windows/Calculator/Calculator.vcxproj [hunk 2]
```diff
@@ -5,18 +5,18 @@
     <CppWinRTOptimized>true</CppWinRTOptimized>
     <CppWinRTRootNamespaceAutoMerge>true</CppWinRTRootNamespaceAutoMerge>
     <MinimalCoreWin>true</MinimalCoreWin>
-    <ProjectGuid>{0a214f22-389d-4fb1-97b0-7ea8a092fcd8}</ProjectGuid>
+    <ProjectGuid>{3e279328-1283-46d4-a5d4-44e691939b29}</ProjectGuid>
```
Comment (kaiguo) [inline samples/Calculator/windows/Calculator/Calculator.vcxproj]: Hmm, so basically we deleted the original project entirely and created a new one? Should we keep the same ProjectGuid as the original project?

Comment (jonthysell) [reply samples/Calculator/windows/Calculator/Calculator.vcxproj]: I guess in a real-world scenario, esp. with an existing app that has already been published to the store, even if you the dev never made any changes to the native code, you wouldn't want to obliterate any "packaging" changes like adding your icons, store metadata, signing certs, etc.

But in this case, this sample existed as a kind of prototype sample before we announced MVP. This PR updates the sample to be a new project using the first stable version we're telling people to really use.

Future updates to the samples, (to new stable versions) should probably not just obliterate everything.

**Pull request merged**